internal/pkg/plugin/sif.go

```diff
@@ -0,0 +1,146 @@
+package plugin
+
+import (
+	"encoding/json"
+
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+
```

Comment:
Extra newline

```suggestion
```

Reply:
With the newline removed sort order should have sif before sylog

Reply:
Thanks @WestleyK!